etl_i2b2/sql_scripts/cms_enr_dstats.sql

```diff
@@ -140,6 +141,29 @@ from ea
 ;
 -- select * from per_bene_mo;
 
+
+
+/* Schroeder writes Tuesday, June 4th, 2019 6:09 PM:
+
+The monthly state buy-in code specifies whether the state contributed to the premiums for parts A/B.  
+It can tell you if the beneficiary had A/B/AB. But it is doesn???t tell you HMO status.
+
+That monthly variable you need is: HMO_IND_XX
+
+So you need MDCR_ENTLMT_BUYIN_IND_XX in (1,2,3,A,B,C)  and  HMO_IND_XX in (0)
+
+Note: technically,  you can include HMO_IND_XX in (0, 4) but I???ve always excluded 
+the ???Fee-for-service participant in case or disease management demonstration project??? (which is the ???4???). 
+https://www.resdac.org/articles/identifying-medicare-managed-care-beneficiaries-master-beneficiary-summary-or-denominator
+
+These variables can be helpful too
+BENE_HI_CVRAGE_TOT_MONS  (number of months of A coverage for that reference year)
+BENE_SMI_CVRAGE_TOT_MONS  (number of months of B coverage for that reference year)
+BENE_HMO_CVRAGE_TOT_MONS  (number of months of HMO coverage for that reference year)
+*/
+
+select * from "&&PCORNET_CDM".enrollment;
```

Comment:
This select should probably be commented out or deleted.

Reply:
removed the select statement in new commit